pro/src/screens/BusinessUnitList/VenueListItem/VenueListItem.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import Icon from '../../../components/layout/Icon'
+import { IBusinessUnitVenue } from '../BusinessUnitList'
+
+import styles from './VenueListItem.module.scss'
+
+interface IVenueListItemProps {
+  venue: IBusinessUnitVenue
+  svg: string
+  siret: string | null
+}
+
+const VenueListItem = ({
+  venue,
+  svg,
+  siret,
+}: IVenueListItemProps): JSX.Element => {
+  return (
+    <div
+      className={`${styles['venue-list-item']} ${
+        venue.isBusinessUnitMainVenue ? styles['reference-venue'] : ''
+      }`}
```

Comment:
Depuis les développement eac, on à acces au helper `cn`.

```suggestion
import cn from 'classnames'
// ...
      className={cn(
        styles['venue-list-item'], 
        { [styles['reference-venue']]: venue.isBusinessUnitMainVenue }
      )}
}
```